src/index.ts

```diff
@@ -1 +1,13 @@
-export * from '.';
\ No newline at end of file
+import { AnalyticsConfig } from './AnalyticsConfig';
+import { AnalyticsEventReporter } from './AnalyticsEventReporter';
+import { AnalyticsEventService } from './AnalyticsEventService';
+
+/**
+ * The Yext Analytics Events SDK
+ * @beta
+ */
+const analytics = (config: AnalyticsConfig): AnalyticsEventService => {
+  return new AnalyticsEventReporter(config);
+};
+
+export default analytics;
```

Comment:
I think we also want to export the AnalyticsConfig and AnalyticsEventService. Seeing warnings about them in the auto generated doc files `etc/analytics.api.md`. Plus, that's what was done before.

https://github.com/yext/analytics/blob/main/src/index.ts#L88